src/ir_Corona.cpp

```diff
@@ -175,8 +173,7 @@ uint8_t IRCoronaAc::getSectionByte(const uint8_t section) {
   // 2 enabled bits shifted 0-2 bits depending on section
   if (section >= 3)
     return 0b10010000 | b;
-  setBits(&b, kHighNibble, kNibbleSize, 0b11 << section);
-  return b;
+  return b | (0b11 << (section+kHighNibble));
```

Comment:
That is not strictly equivalent code.
It probably works fine in the cases we have.

that is, the `setBits()` line is clearing the bits, and then "or"ing them in.
Your implementation is just "or"ing them in.

e.g. if the `0b00010000` bit is set in `b`, the two methods would produce a different result.

Reply:
Thanks for fixing this.